rest/taskana-rest-spring-test/src/main/java/pro/taskana/rest/test/TestWebSecurityConfig.java

```diff
@@ -77,6 +88,7 @@ public LdapAuthoritiesPopulator authoritiesPopulator(
   }
 
   @Bean
+  @Primary
```

Comment:
Why?

Reply:
Because `rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapConfiguration.java -> ldapContextSource()` also creates a bean of type `org.springframework.ldap.core.support.BaseLdapPathContextSource`, and for the tests we want the one from the TestWebSecurityConfig.